thirdparty/patchmatchnet/datasets/data_io.py

```diff
@@ -0,0 +1,93 @@
+"""MVSDataset io utils
+    reference: https://github.com/FangjinhuaWang/PatchmatchNet
+
+"""
+import sys
+from typing import Tuple
+
+import re
+import numpy as np
+
+
+def read_pfm(filename: str) -> Tuple[np.ndarray, float]:
+    """Read a depth map from a .pfm file
+
+    Args:
+        filename: .pfm file path string
+
+    Returns:
+        data: array of shape (H, w, C) representing loaded heat map
```

Comment:
does it load a depth map or a heat map?

Reply:
prefer H,W,C for consistency rather than H,w,C

Reply:
@codyly are you sure this returns (H,W,C)? The inline comment seems to suggest it would return (H,W)

Reply:
nevermind, i see they reshape to (H,W,1) or (H,W,C) at the end